app/routes.js

```diff
@@ -63,8 +63,13 @@ export const routes = (authProvider) => {
     res.render('pages/organisation/template.njk', addContext({ context, user: req.user }));
   }));
 
-  router.post('/organisations/:organisationId', authProvider.authorise(), withCatch(authProvider, async (req) => {
-    logger.info('POST /organisations/:organisationId', req.body);
+  router.post('/organisations/:organisationId', authProvider.authorise(), withCatch(authProvider, async (req, res) => {
```

Comment:
thought we were going to change this route to 
POST - `/organisations/:organisationId/edit`